cluster/ci/config/prow/config.yaml

```diff
@@ -165,6 +165,7 @@ presubmits:
   - name: test_pull_request_origin_extended_image_registry
     agent: jenkins
     always_run: false
+    run_if_changed: '^(pkg/image)|^(test/extended/registry)|^(test/extended/images)|^(test/extended/imageapis)|^(vendor/github.com/docker/distribution)'
```

Comment:
Do you also want `/test/extended/image_ecosystem`?

Reply:
no, that's a different test job.